pandas/io/html.py

```diff
@@ -257,7 +237,7 @@ def _parse_td(self, obj):
 
         Parameters
         ----------
-        obj : node-like
+        obj : an HTML row element
```

Comment:
Is this supposed to be a type or a description? Not sure the exactly terminology but I feel like the type of `node-like` was preferable with this as the description (on subsequent line)

Reply:
changed